modules/meta.analysis/R/single.MA.R

```diff
@@ -13,8 +13,10 @@
 ##' single.MA. This will allow power analysis  to run repeated MA outside of the
 ##' full loop over traits and PFTs.
 ##' @title Single MA
-##' @param data
 ##' @param j.chains number of chains in meta-analysis
+##' @param data data frame generated by jagify function
```

Comment:
should this line come before j.chains so that it is in the same order as the function arguments?

Reply:
Yes, I will make the changes